### ISSUE TITLE
WorkerLostError: Worker exited prematurely (exitcode: -9)

### ISSUE THREAD

**Issue by chrisspen** [celery/celery#1406]
Possibly related to #1294. The error email I get from Celery is fairly minimal and unhelpful in diagnosing the issue:

```
Task tasks.myapp.mytask with id bffae5cb-c203-4258-b13b-2d7f8be167ac raised exception:
"WorkerLostError('Worker exited prematurely (exitcode: -9).',)"


Task was called with args: [162L] kwargs: {}.

The contents of the full traceback was:

Traceback (most recent call last):
  File "/usr/local/myproject/.env/lib/python2.7/site-packages/billiard/pool.py", line 950, in _join_exited_workers
    lost_ret, ))
WorkerLostError: Worker exited prematurely (exitcode: -9).
```

I tried updating billiard to 2.7.3.28 and restarting Celery but the error still occurs.

Please let me know if there's any further info I can provide to help fix this problem. I don't see anything listed in my Celery log.

**Comment by FirojNeosoft** [comment]
Hi,
I got below error while running long task.
Task handler raised error: WorkerLostError('Worker exited prematurely: signal 9 (SIGKILL).')

I used below tools-
celery==4.3.0, amqp==2.5.0, RabbitMQ 3.3.5, AWS EC2 instance as server

I ran 2 long tasks. I got same error for both tasks.
Then, I converted server type medium to large for EC2 instance, and I got success for one of the 2 tasks. But still one task is not successful due to workerLostError.
